seeq/addons/email/condition_monitor/_email_builder.py

```diff
@@ -19,7 +21,8 @@ def fill_in_template(template, capsule, job):
         for to_replace, prop in capsule_substitutions:
             replacement = str(capsule[prop]) if prop in capsule else '{Capsule property not found}'
             if prop in ['Capsule Start', 'Capsule End']:
```

Comment:
minor, not important: 'Capsule Start', 'Capsule End' can be extracted in some constant and reused.